eng/pipelines/common/download-specific-artifact-step.yml

```diff
@@ -12,10 +12,10 @@ steps:
     inputs:
       buildType: specific
       project:  'internal' # 'internal' or 'public'
-      pipeline: 'superpmi-replay'
```

Comment:
Is this file only used for testing?

Reply:
Yes, this is used to skip building coreclr/libraries and download these from previous runs. This helps in fast testing during pipeline development.